src/app/components/user-info/page-header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../../styles/user-info.css';
+
+const PageHeader = (props) => {
+  const user = props.user;
+  return (
+    <>
+      <div className="header">
+        <div className="user-details-div margin-left">
+          <img className="user-avatar" src={user.avatarUrl} alt={user.name}></img>
```

Comment:
@bhoomika08 purpose of alt?

Reply:
@gurpreet013 It's an alternative text for an image if the image to be displayed is not found.